app/assets/javascripts/discourse/app/components/user-notification-schedule-day.js

```diff
@@ -0,0 +1,12 @@
+import Component from "@ember/component";
+import I18n from "I18n";
+import discourseComputed from "discourse-common/utils/decorators";
+
+export default Component.extend({
+  tagName: "",
+
+  @discourseComputed("day")
```

Comment:
we have an i18n helper which would allow you to just do:

`dayLabel: i18n("day", "user. notification_schedule.%@")`